### PR TITLE
[DOCS] BitmapFont

### PR DESCRIPTION
**Pull request by diowonderland**

### CODE REVIEW

**Review by cbscribe — CHANGES_REQUESTED** [review]

**Comment by diowonderland** [comment]
Updated!

File: doc/classes/BitmapFont.xml [hunk 1]
```diff
@@ -23,7 +25,7 @@
 			<argument index="4" name="advance" type="float" default="-1">
 			</argument>
 			<description>
-				Add a character to the font, where [i]character[/i] is the unicode value, [i]texture[/i] is the texture index, [i]rect[/i] is the region in the texture (in pixels!), [i]align[/i] is the (optional) alignment for the character and [i]advance[/i] is the (optional) advance.
+				Adds a character to the font, where [i]character[/i] is the unicode value, [i]texture[/i] is the texture index, [i]rect[/i] is the region in the texture (in pixels!), [i]align[/i] is the (optional) alignment for the character and [i]advance[/i] is the (optional) advance.
```
Comment (Noshyaar) [inline doc/classes/BitmapFont.xml]: maybe [code] instead of [i]?

File: doc/classes/BitmapFont.xml [hunk 2]
```diff
@@ -136,24 +144,31 @@
...
+				Sets the total font height (ascent plus descent) in pixels.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="ascent" type="float" setter="set_ascent" getter="get_ascent">
+			Ascent (number of pixels above the baseline).
 		</member>
 		<member name="chars" type="PoolIntArray" setter="_set_chars" getter="_get_chars">
+			The chars in the BitmapFont.
```
Comment (Noshyaar) [inline doc/classes/BitmapFont.xml]: 'characters' would be clearer imo

**Comment by diowonderland** [comment]
Updated!

**Review by cbscribe — APPROVED** [review]

**Comment by cbscribe** [comment]
Thanks for your contribution!

**Pull request merged**